app/Jobs/GenerateHashValue.php

```diff
@@ -122,10 +127,8 @@ protected function getHashValue(Image $image)
             $script = config('biigle.hash_value_generator');
 
             try {
-                $outputPath = $this->getOutputJsonPath($image);
                 $inputPath = $this->createInputJson($image, $imageByteString);
-                $output = $this->python("{$script} {$inputPath} {$outputPath}");
-                $hashValue = $this->decodeOutputJson($outputPath);
+                $hashValue = $this->python("{$script} {$inputPath}")[0];
```

Comment:
```suggestion
                $hashValue = $this->python("{$script} {$inputPath}");
```
The `end($lines)` of `python()` already returns the last array element.